backend/src/common/util/workflow.go

```diff
@@ -162,9 +162,11 @@ func (w *Workflow) HasScheduledWorkflowAsParent() bool {
 }
 
 func (w *Workflow) GetSpec() *Workflow {
```

Comment:
Can you rename this function as well, since it's returning the whole workflow, not just the spec

Reply:
done. hopefully getworkflowspec can reduce the confusion.